resolwe/test/testcases/process.py

```diff
@@ -240,8 +240,23 @@ def tearDown(self):
 
         super().tearDown()
 
+        # Check test outcome to prevent failing the test twice.
```

Comment:
🙂 , I was thinking of the same thing today when I watched the output of process failures.

Please, add a CHANGELOG entry. Perhaps a more succinct explanation would be something like:
Do not report additional failure when testing a tagged process errors or fails.